Source/JavaScriptCore/runtime/ImportMap.h

```diff
@@ -43,6 +43,7 @@ class ImportMap final : public RefCounted<ImportMap> {
         SpecifierMap m_map;
     };
     using Scopes = Vector<ScopeEntry>;
+    using Integrity = HashMap<URL, String>;
```

Comment:
I think it's confusing for this hash map to be called `Integrity`. Maybe `IntegrityMap`?

Reply:
Renamed!